diff --git a/letsencrypt_glassfish.pl b/letsencrypt_glassfish.pl
index e89e947..7d09503 100755
--- a/letsencrypt_glassfish.pl
+++ b/letsencrypt_glassfish.pl
@@ -98,9 +98,9 @@ my $execute_cmd = $false; # Change this to actually do something!
 # Otherwise, there's nothing for which a cert can be generated.
 #
 my @domains = qw/
-   domainx.net
-   oc-stuff.com 
-   alphaforensic.com
+   m79.net
+   oc-water.com
+   unixforensic.com
 /;
 
 # ----
